rust/processor/migrations/2023-09-01-231248_events_v2/down.sql

```diff
@@ -1,8 +1 @@
 -- This file should undo anything in `up.sql`
```

Comment:
`down.sql` not needed?

Reply:
nah- many of these shouldn't have had a `down` to begin with

Reply:
when should it not have a `down`?